src/router.js

```diff
@@ -163,6 +156,14 @@ function Router({
     return createResponse(statusCode, body, headers, route && route.path, requestPath)
   }
 
+  const addBatchRoute = (path, config) => {
+    routes.push({
+      method: 'POST',
+      path,
+      handler: batchHandler.bind(null, { route, config })
+    })
+  }
+  
```

Comment:
Moved addBatchRoute lower so that the route method was declared before.

Removed extractPathParameters as I believe the route function itself handles that.